modules/doc/content/applications/thermal_hydraulics.md

```diff
@@ -0,0 +1,132 @@
+# Thermal Hydraulics Applications
+
+MOOSE provides versatile, general-purpose thermal-hydraulics applications. These applications solve for mass, momentum, and energy conservation in multicomponent, multiphase flows using incompressible, weakly-compressible, or fully compressible formulations for steady-state or transients in lumped parameters and/or multidimensional (1, 2, or full 3D) geometries.
```

Comment:
```suggestion
MOOSE provides versatile, general-purpose thermal-hydraulics applications. These applications solve for mass, momentum, energy, and advected species conservation in multicomponent, multiphase flows using incompressible, weakly-compressible, or fully compressible formulations for steady-state or transients in lumped parameters and/or multidimensional (1, 2, or full 3D) geometries.
```

Reply:
MOOSE itself is general purpose, the provided thermal-hydraulics applications are each quite specific. I propose the following:

MOOSE provides multiple thermal-hydraulic applications for varied purposes. Cumulatively, these applications solve for mass, momentum, energy, and advected species conservation in multicomponent, multiphase flows using incompressible, weakly-compressible, or fully compressible formulations for steady-state or transients in lumped parameters and/or multidimensional (1, 2, or full 3D) geometries.